### DISCUSSION TITLE
Problem with the index file.

### DISCUSSION THREAD

**Discussion by donatienoussa** [symfony/symfony#49866] [Q&A]
Hi Everybody. I need your help. 
I create my website with Symfony 6. 
The problem is that: my index.php file is in the public directory but my host ask me to put it in the htdocs. How to change, in the host(LWS), where my index file is ?

**Comment by ker0x** [comment]
Hello, it is well explained in the documentation:

[Override the Public Directory](https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory)
